packages/web/src/scss/components/Card/_CardLogo.scss

```diff
@@ -0,0 +1,25 @@
+@use 'theme';
+
+.CardLogo {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    border: theme.$logo-border-width theme.$logo-border-style theme.$logo-border-color;
+    border-radius: theme.$logo-border-radius;
+    background-color: theme.$logo-background-color;
+}
+
+.Card--vertical > .CardLogo {
+    grid-area: media;
+    place-self: end end;
```

Comment:
```suggestion
    place-self: end;
```

isn't this enough?

Reply:
I'm a little bit scared, but let's go with a single value.